src/test/java/org/folio/circulation/domain/policy/library/KeepCurrentStrategyTest.java

```diff
@@ -4,33 +4,65 @@
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeConstants;
 import org.joda.time.DateTimeZone;
-import org.junit.Assert;
 import org.junit.Test;
 
 import static org.folio.circulation.domain.policy.library.ClosedLibraryStrategyUtils.END_OF_A_DAY;
+import static org.joda.time.DateTimeZone.UTC;
+import static org.joda.time.DateTimeZone.forOffsetHours;
+import static org.junit.Assert.assertEquals;
+
+import java.util.stream.IntStream;
 
 public class KeepCurrentStrategyTest {
 
   @Test
   public void testKeepCurrentDateStrategy() {
-    ClosedLibraryStrategy keepCurrentStrategy = new KeepCurrentDateStrategy(DateTimeZone.UTC);
+    ClosedLibraryStrategy keepCurrentStrategy = new KeepCurrentDateStrategy(UTC);
     DateTime requestDate =
       new DateTime(2019, DateTimeConstants.JANUARY, 1, 0, 0)
```

Comment:
Personally, I think we might as well statically import `January` as well, it might mean that initialisation can fit on one line?

Reply:
Yes, done